### ISSUE TITLE
UI flow

### ISSUE THREAD

**Issue by s3687636** [nguyenducminhtam/sepm-snapie#28]
basic and simple UI that friendly to target users.
@LcNguyn

**Comment by LcNguyn** [comment]
**Here are the Balsamiq Mockups.**
These are basic layouts created in order for our team to have clearer views of our app as well as starting our frontend coding. 
_Note: Quiz and Profile/Rewards mockup will be added soon._
![Main](https://user-images.githubusercontent.com/37359318/77438249-28bd8180-6e18-11ea-9aa6-a0361a6a00f1.PNG)
![Explore](https://user-images.githubusercontent.com/37359318/77438255-29eeae80-6e18-11ea-9150-73ca747b8f8e.PNG)
![Book](https://user-images.githubusercontent.com/37359318/77438259-2a874500-6e18-11ea-8d51-b46d6cf43951.PNG)

**Comment by LcNguyn** [comment]
P/s: Explore: Representative Image based on each word will be added to mock up later.

**Comment by s3687636** [comment]
you can build on react-native components for them now. 
note: must be components for react-native to be rendered.
you can use dummy data for now (const json object).

**Comment by LcNguyn** [comment]
> you can build on react-native components for them now.
> note: must be components for react-native to be rendered.
> you can use dummy data for now (const json object).

this is just some mockup :)) the UI needs more than this so I think I cant code them, at least for now. 
I think it would be more convenient if you can add the buttons relevant to the functionality you are testing so that I won't have to do that again. Or I can try to do it simultaneously with my tasks. :))) what do u think?

**Comment by trungduong0103** [comment]
Some comments on the UI mockups:
Is our first screen the main screen ? when we first open our application we will see this screen right ? No log in no sign up. If this is the case, I don't think we should put our logo here, it's entirely redundant. Moreover, the main screen has 5 list items, I don't think that's a good design. My opinion is to put two big icons that has a floating effect over the main screen. The first icon is a compass, that represent the Explore screen, the second one is the open book, represents the view screen. Quiz can also be added under those two icons but not preferred. 
Lastly, Me and Settings should be put as a bottom navigation, showing all options in one huge chunk looks cluttered.
Just my idea.

**Comment by trungduong0103** [comment]
> Some comments on the UI mockups:
> Is our first screen the main screen ? when we first open our application we will see this screen right ? No log in no sign up. If this is the case, I don't think we should put our logo here, it's entirely redundant. Moreover, the main screen has 5 list items, I don't think that's a good design. My opinion is to put two big icons that has a floating effect over the main screen. The first icon is a compass, that represent the Explore screen, the second one is the open book, represents the view screen. Quiz can also be added under those two icons but not preferred.
> Lastly, Me and Settings should be put as a bottom navigation, showing all options in one huge chunk looks cluttered.
> Just my idea.

@LcNguyn ideas ?

**Comment by trungduong0103** [comment]
<img width="435" alt="Screen Shot 2020-03-25 at 15 34 21" src="https://user-images.githubusercontent.com/15820443/77517227-59e89100-6eae-11ea-905b-27de2c5a3f6d.png">

**Comment by LcNguyn** [comment]
> Some comments on the UI mockups:
> Is our first screen the main screen ? when we first open our application we will see this screen right ? No log in no sign up. If this is the case, I don't think we should put our logo here, it's entirely redundant. Moreover, the main screen has 5 list items, I don't think that's a good design. My opinion is to put two big icons that has a floating effect over the main screen. The first icon is a compass, that represent the Explore screen, the second one is the open book, represents the view screen. Quiz can also be added under those two icons but not preferred.
> Lastly, Me and Settings should be put as a bottom navigation, showing all options in one huge chunk looks cluttered.
> Just my idea.

Logo position: Agree. I am considering putting it at the loading screen when the user starts the app.

Menu layout: I have considered doing that at first since it's our team's first imagination. However, the problem is the layout design. The screen will have lots of blank spaces which we need to design a super interesting background behind in order to get rid of users' boredom which is quite risky. So I think it's better to keep this menu (menu inspired by https://www.pinterest.com/pin/238690848988217830/
) 
Btw, I have reduced the number of list items from 5 to 4 (might be reduced to 3). I have put the Setting item inside Me item since I got the idea from some IOS app.

**Comment by LcNguyn** [comment]
Update: Added Quiz, Profile, Setting MockUps. Logo might be finished by tomorrow. 
If the Mockups are all reviewed and agreed on by our team. I will start going to details.
![Quiz_Tradi](https://user-images.githubusercontent.com/37359318/77547740-2aeb1300-6ee0-11ea-8956-535fa62b2d16.PNG)
![Quiz_VoiceCam](https://user-images.githubusercontent.com/37359318/77547864-540ba380-6ee0-11ea-83a6-21dec2d42e4f.PNG)
![Result](https://user-images.githubusercontent.com/37359318/77547883-5a018480-6ee0-11ea-93d3-9fbbea9131b0.PNG)
![Profile](https://user-images.githubusercontent.com/37359318/77547771-33dbe480-6ee0-11ea-9e91-a2d805554292.PNG)

**Comment by thienhieu215** [comment]
I agree with Trung  about the layout of home screen. However, the profile and setting buttons can be changed to icons and put them at the corners.
The only way to access the history screen is to go to the explore camera screen first then click that button at the top corner, am I right?

**Comment by LcNguyn** [comment]
We are trying to reduce the icons (that are not labeled) as much as we can. So we cannot use to much icon. Everything must be clear and understandable as much as possible

Nguyen Tuan Loc

**Comment by thienhieu215** [comment]
I think the setting and profile icons are simple and easy to understand. How about the history screen I mentioned above?

**Comment by trungduong0103** [comment]
looks good to me
